invenio.cfg

```diff
@@ -341,3 +341,26 @@ RDM_CUSTOM_FIELDS = [
 # Search query of recent uploads
 # Defaults to newest records search
 ZENODO_FRONTPAGE_RECENT_UPLOADS_QUERY = "/api/records?sort=newest&size=10"
+
+
+# Journal CF
+
+from invenio_rdm_records.contrib.journal.custom_fields import (
+    JOURNAL_NAMESPACE,
+    JOURNAL_CUSTOM_FIELDS,
+    JOURNAL_CUSTOM_FIELDS_UI
+)
+
+RDM_CUSTOM_FIELDS_UI = [
+    JOURNAL_CUSTOM_FIELDS_UI
+]
+
+RDM_CUSTOM_FIELDS = [
+    *RDM_CUSTOM_FIELDS,
+    *JOURNAL_CUSTOM_FIELDS,
```

Comment:
Same as below

Reply:
Nice catch, it was a leftover from testing codemeta + journal at the same time.
I will move it!